schedule/schedule-appengine/src/main/i18n/locale-en.json

```diff
@@ -34,6 +34,7 @@
   "AGENDA": "Agenda",
   "AGENDA_DESCRIPTION": "View upcoming booked Activities",
   "AMOUNT": "Amount",
+  "APPLY_VALUES": "Apply",
```

Comment:
At some point in the distant future we might want to come up with a naming standard